### PR TITLE
add support for borderTopLeftRadius and the three other one-corner-on…

### PR DESCRIPTION
**Pull request by zholobov**
…ly border radius props to ReactImageManager

### CODE REVIEW

File: ReactWindows/ReactNative/Views/Image/ReactImageManager.cs [hunk 1]
```diff
@@ -161,15 +161,53 @@ public void SetSource(Border view, JArray sources)
...
+        [ReactPropGroup(
+            ViewProps.BorderRadius,
+            ViewProps.BorderTopLeftRadius,
+            ViewProps.BorderTopRightRadius,
+            ViewProps.BorderBottomLeftRadius,
+            ViewProps.BorderBottomRightRadius)]
+        public void SetBorderRadius(Border view, int index, double radius)
         {
-            view.CornerRadius = new CornerRadius(radius);
+            var cornerRadius = view.CornerRadius;
```
Comment (rigdern) [inline ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: >view.CornerRadius [](start = 31, length = 17)

Can `view.CornerRadius` ever be null? You can see `ReactViewManager` uses this expression:

```
var cornerRadius = border.CornerRadius == null ? new CornerRadius() : border.CornerRadius;
```

File: ReactWindows/ReactNative/Views/Image/ReactImageManager.cs [hunk 2]
```diff
@@ -161,15 +161,53 @@ public void SetSource(Border view, JArray sources)
...
+
         /// <summary>
         /// The border radius of the <see cref="ReactRootView"/>.
         /// </summary>
         /// <param name="view">The image view instance.</param>
+        /// <param name="index">The prop index.</param>
         /// <param name="radius">The border radius value.</param>
-        [ReactProp("borderRadius")]
-        public void SetBorderRadius(Border view, double radius)
+        [ReactPropGroup(
```
Comment (matthargett) [inline ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: modify WPF implementation to match as well. test prop in Playground and provide code+screenshot for both UWP and WPF.

Comment (zholobov) [reply ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: @matthargett I have another PR ready to be submitted which fixes a bug in borderRadius and which has WPF part updated too. Is it okay if you approve this PR and I will do the Playground screenshots and code as part pf the next PR?

File: ReactWindows/ReactNative/Views/Image/ReactImageManager.cs [hunk 3]
```diff
@@ -161,15 +161,53 @@ public void SetSource(Border view, JArray sources)
             }
         }
 
+        /// <summary>
+        /// Enum values correspond to positions of prop names in ReactPropGroup attribute
+        /// applied to <see cref="SetBorderRadius(Border, int, double)"/>
+        /// </summary>
+        private enum Radius
```
Comment (rozele) [inline ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: BEGIN NITPICK

In general I don't know why we add private enums when ints work just fine.

END NITPICK

File: ReactWindows/ReactNative/Views/Image/ReactImageManager.cs (hunk 2 above)
Comment (rozele) [reply ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: I'd prefer to see a contribution upstream for a `border*Radius` example in RNTester. Does not need to be done for this repo.

Comment (zholobov) [reply ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: ReactNative already has examples of border*Radius at https://github.com/facebook/react-native/edit/master/RNTester/js/BorderExample.js see styles border6, border9, border10, border14.

Comment (rozele) [reply ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: Yes - but not for Image

Comment (zholobov) [reply ReactWindows/ReactNative/Views/Image/ReactImageManager.cs]: @rozele PR into ReactNative for boredr*Radius test for Image seems to be in process of landing [here](https://github.com/facebook/react-native/pull/20264).

**Pull request merged**